OpenSim/Common/TRCFileAdapter.cpp

```diff
@@ -194,22 +198,37 @@ TRCFileAdapter::extendRead(const std::string& fileName) const {
             } // otherwise the value will remain NaN (default)
             ++ind;
         }
-
+        markerData[rowNumber] = row_vector;
         // Column 1 is time.
-        table->appendRow(std::stod(row.at(1)), std::move(row_vector));
-
+        times[rowNumber] = std::stod(row.at(1));
+        rowNumber++;
+        if (std::remainder(rowNumber, last_size) == 0) {
```

Comment:
Could this use `rowNumber == last_size` instead? The remainder would be 0 if `rowNumber` is 50 and `last_size` is 10.

Reply:
By construction rowNumber will always be less than last_size but I can change the test to == if it makes it more clear.

Reply:
Yes, I think it is more clear. By using `remainder`, a maintainer might think that `rowNumber` could go greater.